tools/scylla-nodetool.cc

```diff
@@ -2644,6 +2645,241 @@ void table_stats_operation(scylla_rest_client& client, const bpo::variables_map&
+        add_tree_to_statuses_lists(table, res_el);
+    }
+
+    table.print();
+}
+
+static void print_stats_list(const rjson::value& res) {
+    auto stats = res.GetArray();
+    Tabulate table;
+    table.add("task_id", "type", "kind", "scope", "state", "sequence_number", "keyspace", "table", "entity");
+    for (auto& element : stats) {
+        const auto& s = element.GetObject();
+
+        table.add(rjson::to_string_view(s["task_id"]),
+                rjson::to_string_view(s["type"]),
+                rjson::to_string_view(s["kind"]),
+                rjson::to_string_view(s["scope"]),
+                rjson::to_string_view(s["state"]),
+                s["sequence_number"].GetUint64(),
+                rjson::to_string_view(s["keyspace"]),
+                rjson::to_string_view(s["table"]),
+                rjson::to_string_view(s["entity"]));
+    }
+    table.print();
+}
+
+void tasks_abort_operation(scylla_rest_client& client, const bpo::variables_map& vm) {
+    if (!vm.contains("id")) {
+        throw std::invalid_argument("required parameter is missing: id");
+    }
+    auto id = vm["id"].as<sstring>();
+
+    auto res = client.post(format("/task_manager/abort_task/{}", id));
+    fmt::print("OK");
+}
+
+void tasks_list_operation(scylla_rest_client& client, const bpo::variables_map& vm) {
+    if (!vm.contains("module")) {
+        throw std::invalid_argument("required parameter is missing: module");
+    }
+    auto module = vm["module"].as<sstring>();
+    std::unordered_map<sstring, sstring> params;
+    if (vm.contains("keyspace")) {
+        params["keyspace"] = vm["keyspace"].as<sstring>();
+    }
+    if (vm.contains("table")) {
+        params["table"] = vm["table"].as<sstring>();
+    }
+    if (vm.contains("internal")) {
+        params["internal"] = "true";
+    }
+    int period = vm["period"].as<int>();
+    int iterations = vm["iterations"].as<int>();
+
+    auto res = client.get(format("/task_manager/list_module_tasks/{}", module), params);
+    print_stats_list(res);
+
+    for (auto i = 0; i < iterations; ++i) {
+        sleep(period);
+        auto res = client.get(format("/task_manager/list_module_tasks/{}", module), params);
+        fmt::print("\n\n");
+        print_stats_list(res);
+    }
+}
+
+void tasks_modules_operation(scylla_rest_client& client, const bpo::variables_map& vm) {
+    auto res = client.get("/task_manager/list_modules");
+
+    for (auto& x: res.GetArray()) {
+        fmt::print("{}\n", rjson::to_string_view(x));
+    }
+}
+
+void tasks_set_ttl_operation(scylla_rest_client& client, const bpo::variables_map& vm) {
+    if (!vm.contains("ttl")) {
+        throw std::invalid_argument("required parameter is missing: ttl");
+    }
+    auto new_ttl = vm["ttl"].as<uint32_t>();
+    std::unordered_map<sstring, sstring> params = {{ "ttl", fmt::format("{}", new_ttl) }};
+
+    auto res = client.post("/task_manager/ttl", std::move(params));
+
+    fmt::print("Set ttl value to {}\n", new_ttl);
```

Comment:
Again, I think we should try to follow the UNIX philosophy here, where commands don't print anything on success.